### PR TITLE
Adding integration tests for the browser usage

### LINKED ISSUE #679: Browser support does not seem to be correctly documented
**Description**

The document says that the library can be used in a browser using this script:

```html
<script src="node_modules/@gitbeaker/browser/index.js" />
<script>
  const { Gitlab } = gitlab;
```

However, there is no `node_modules/@gitbeaker/browser/index.js` in the `@gitbeaker/browser` dependency.

There is a `node_modules/@gitbeaker/browser/dist/index.js` but it does not seem to export any `gitlab` object.

**Possible solutions**

I guess the documentation needs an update on this.

Thanks!

### LINKED ISSUE #680: Add browser distribution integration tests
Could do something based on a snippet like below

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=1, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  <script src="node_modules/@gitbeaker/browser/dist/index.js">
  </script>
  <script>
    const { Gitlab } = gitbeaker.default;
  </script>
</body>
</html>
```
You need to add `.default` at least that how it is in Chrome.
Not sure default is a great experience 😓

_Originally posted by @jetersen in https://github.com/jdalrymple/gitbeaker/issues/679#issuecomment-609846617_

### PR DESCRIPTION
**Pull request by jdalrymple**
fixes #680
fixes #679

### CODE REVIEW

**Comment by jdalrymple** [comment]
@jetersen Mind giving this a quick look over?

**Comment by jetersen** [comment]
@jdalrymple if you are fixing #698 and #686 that should also affect this PR?

also you should update the PR description to get the auto close working

fixes #680
fixes #679

File: packages/gitbeaker-browser/test/integration/assets/test-import.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=1, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script src="../../../dist/index.js">
+  </script>
+</body>
+</html>
```
Comment (jetersen) [inline packages/gitbeaker-browser/test/integration/assets/test-import.html:12]: final new line 👮‍♂️

File: packages/gitbeaker-browser/test/integration/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
+
+describe('Browser Import', () => {
+  it('should import the Gitbeaker library though the global gitbeaker', async () => {
+    await page.goto(`file://${path.resolve(__dirname, 'assets', 'test-import.html')}`);
```
Comment (jetersen) [inline packages/gitbeaker-browser/test/integration/index.ts:5]: neat 👍

**Comment by jdalrymple** [comment]
> @jdalrymple if you are fixing #698 and #686 that should also affect this PR?
> 
> also you should update the PR description to get the auto close working
> 
> fixes #680
> fixes #679

Im doing this in pieces :)

**Comment by jdalrymple** [comment]
:rocket: PR was released in [`18.1.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/18.1.0) :rocket:

**Pull request merged**